src/kixi/spec/conformers.clj

```diff
@@ -289,16 +289,16 @@
   (-regex? #"^http[s]?:\/\/([^:\/\s]+)((\/\w+)*\/)([\w\-\.]+[^#?\s]+)(.*)?(#[\w\-]+)?$"))
 
 (def url?
-  (s/with-gen
-    (s/conformer url? identity)
-    (fn []
-      (gen/fmap fmt-url
-                (gen/tuple
-                 (gen/boolean)
-                 (gen/vector (gen/such-that #(< 1 (count %))  (gen/string-alphanumeric)) 2 5)
-                 (gen/vector (gen/such-that #(< 1 (count %))  (gen/string-alphanumeric)) 2 5)
-                 (gen/one-of [(gen/such-that #(< 1 (count %)) (gen/string-alphanumeric))
-                              (gen/return nil)]))))))
+  (let [not-blank #(not (clojure.string/blank? %))]
+    (s/with-gen
+      (s/conformer -url? identity)
+      #(gen/fmap fmt-url
+                 (gen/tuple
+                  (gen/boolean)
+                  (gen/vector (gen/such-that not-blank  (gen/string-alphanumeric)) 2 5)
```

Comment:
just straight up "not-empty":
kixi.event-fixer> (not-empty nil)
nil
kixi.event-fixer> (not-empty "")
nil
kixi.event-fixer> (not-empty "a")
"a"

Reply:
O, your worried about " ", but this won't stop "a b" will it?

Reply:
Doesn't account for whitespace